lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,135 @@
+"""Edx footer API
```

Comment:
Nitpick: Eventually, this won't just be the footer.  Maybe call it the "branding" API?